benchmarks/kernels/weight_shapes.py

```diff
@@ -40,4 +40,10 @@
         ([8192, 57344], 1),
         ([28672, 8192], 0),
     ],
+    "meta-llama/Llama-3.1-405b-hf": [
+        ([16384, 18432], 1),
+        ([16384, 16384], 0),
+        ([16384, 106496], 1),
+        ([53248, 16384], 0),
+    ],
```

Comment:
nit : for big models, I have found it useful to have their realistic TPn counter-parts also (e.g. for the 70B case, add a 70B-TP4 case). That way we can just list that version in the 1GPU  model benchmarking.

Reply:
You mean so you can list it as a string as opposed to using the `--tp-sizes` args?